python/cuml/metrics/__init__.py

```diff
@@ -33,5 +33,6 @@
 from cuml.metrics.confusion_matrix import confusion_matrix
 from cuml.metrics.cluster.entropy import cython_entropy as entropy
 from cuml.metrics.pairwise_distances import pairwise_distances, \
-    PAIRWISE_DISTANCE_METRICS
+    sparse_pairwise_distances, PAIRWISE_DISTANCE_METRICS, \
```

Comment:
Nitpick: Can we put these on two separate lines? I know we aren't doing this universally yet but  Cupy does this and I've found it really increases maintainability and readability.